app/src/main/java/org/hackillinois/android/view/LoginActivity.kt

```diff
@@ -93,7 +93,7 @@ class LoginActivity : AppCompatActivity() {
 
             override fun onResponse(call: Call<Roles>, response: Response<Roles>) {
                 if (response.isSuccessful &&
-                        response.body()?.roles?.contains("Staff") == true) {
+                        response.body()?.isStaff() == true) {
```

Comment:
`== true`? 😤

Reply:
it's because the result is nullable and it's cleaner than (response.body?.isStaff() ?: false) lol